packages/runtime/container-runtime/src/blobManager.ts

```diff
@@ -644,9 +591,6 @@ export class BlobManager extends TypedEventEmitter<IBlobManagerEvents> {
 			return;
 		}
 		if (!this.runtime.connected) {
-			if (entry.status === PendingBlobStatus.OnlinePendingUpload) {
-				this.transitionToOffline(localId);
-			}
 			// we are probably not connected to storage but start another upload request in case we are
 			entry.uploadP = this.retryThrottler
```

Comment:
can you file an issue for this spot. i'm not sure what we should do here. part of me thinks we should retry forever in all cases, and the application can use the abort signal if it wants to cancel. whatever we decided i think the current split in behavior is odd, and won't be obvious to developers